walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/travel.html

```diff
@@ -1,13 +1,44 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>Jonathan's Travels</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <title>Jonathan's Travels</title>
+  <link rel="stylesheet" href="style.css">
   </head>
   <body>
     <div id="content">
-      <h1>Jonathan's Travels</h1> 
+      <h1>Jonathan's Travels</h1>  
+      <h2>Barcelona</h3>
```

Comment:
nit: mismatching tags for opening and closing. Also, same question about if the headings should be different sizes.